### ISSUE TITLE
why loss is nan when i train in GPU，and loss is normal train in cpu 

### ISSUE THREAD

**Issue by Zev-Yin** [thtrieu/darkflow#963]
python flow --model cfg/tiny-yolo-voc-3c.cfg --train --dataset VOCdevkit/VOC2012/JPEGImages --ann
otation VOCdevkit/VOC2012/Annotations --lr 0.001 --batch 32 --gpu 0.6
this loss is nan
python flow --model cfg/tiny-yolo-voc-3c.cfg --train --dataset VOCdevkit/VOC2012/JPEGImages --ann
otation VOCdevkit/VOC2012/Annotations --lr 0.001 --batch 32
this loss is normal

**Comment by Zev-Yin** [comment]
i already find why it wrong.loss calculate in train.py
loss = tf.pow(adjusted_net_out - true, 2)
modified to
loss=(adjusted_net_out - true)*(adjusted_net_out - true)
then it works
